packages/terra-time-input/src/TimeUtil.js

```diff
@@ -269,6 +288,141 @@ class TimeUtil {
 
     return variant;
   }
+
+  /**
+   * Is the Time Input operating as a 12-hour variant?
+   * @param {Object} The TimeInput props.
+   * @returns {Boolean} True iff the locale allows a 12-hour time and the variant is 12-hour time.
```

Comment:
```suggestion
   * @returns {Boolean} True if the locale allows a 12-hour time and the variant is 12-hour time.
```

Reply:
Not a typo, _iff_ meaning _If, and only if_.